README.md

```diff
@@ -0,0 +1,22 @@
+## Introduction                                                                                                     
+This toolset helps to split serial communication interface into 2 ACM ports. 
```

Comment:
First of all, it`s not an abstract toolset. It`s firmware for the particular MCU.
Probably, we need to include a description of supported dev boards.